handlers/pipeline.go

```diff
@@ -136,6 +136,50 @@ func PipelineGet(c echo.Context) error {
 	return c.String(http.StatusNotFound, errPipelineNotFound.Error())
 }
 
+// PipelineDelete accepts a pipeline id and deletes it from the
+// store. It also removes the binary inside the pipeline folder.
+func PipelineDelete(c echo.Context) error {
+	pipelineIDStr := c.Param("pipelineid")
```

Comment:
There is no need for this variable to exist. `c.Param` will return an empty string if the parameter is not found making Atoi fail gracefully. So this can be removed and inlined into Atoi.

Reply:
I thought so too initially but after seeing this pattern used in other handler functions, I decided to stick with it. :smile: